docs/intro-to-dune/index.md

```diff
@@ -176,6 +169,42 @@ src/counter
 └─ index.html
 ```
 
+## Structure of `_build` directory
```

Comment:
I added a small section about the structure of the `_build` directory. I plan to refer to this section in later parts of the book.